src/components/Admin/EditChallenge.tsx

```diff
@@ -60,10 +60,10 @@ const EditChallenge: FC<Props> = ({ paths, mode, title }) => {
 
   const isLoading = isCreating || isUpdating;
 
-  const currentPaths = useMemo(() => [...paths, { name: challengeId ? 'edit' : 'new', real: false }], [
```

Comment:
@atstojanov Any idea why those changes are poping up here?

Reply:
Because of updating prettier.